lib/src/license_detection/license.dart

```diff
@@ -4,6 +4,10 @@
 
 part of 'license_detector.dart';
 
+/// Regular expression to match the common pattern of `Copyright (c) yyyy[-yyyy] <name>.`
+final _copyrightRegExp =
+    RegExp(r'^copyright.+\d{4}.{0,100}$', caseSensitive: false);
```

Comment:
```dart
    RegExp(r'^copyright.{0,5}\d{4}.{0,100}$', caseSensitive: false);
```
isn't `.+` a bit risky? how about limiting it to 5 characters

or maybe:
```dart
    RegExp(r'^copyright\s+(?:(c)|©)\s+\d{4}.{0,100}$', caseSensitive: false);
```

Reply:
Updated with the later, using a `\(c\)` to fix parens.